docs/general/status.rst

```diff
@@ -21,14 +21,21 @@ who have already installed it as part of the pilot program).
 That said, the project is entirely open source, and you can install it
 independently if you want to get a feel for how it all works. 
 
-If you're a journalist or news organization trying to decide if this is right for you, please note that SecureDrop Workstation should only be used
+If you're a journalist or news organization trying to decide if this is right for you,
+please note that SecureDrop Workstation should only be used
 by organizations that meet the following criteria:
 
 * You have an existing SecureDrop Server in place
 * You have fewer than 500 Sources on the Server
 * You typically only interact with .pdf, .doc, and .jpg files via your
-  SecureDrop
+  SecureDrop (as opposed to large datasets or complex file formats, which are not
+  yet supported--see below)
```

Comment:
```suggestion
  yet supported — see below)
```

Not sure if we've used an em dash elsewhere in the docs, but I think that's probably more ideal than two dashes here.